Podverse/AppDelegate.swift

```diff
@@ -90,6 +92,21 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         UINavigationBar.appearance().titleTextAttributes = [NSForegroundColorAttributeName: UIColor.white, NSFontAttributeName: UIFont.boldSystemFont(ofSize: 17.0)]
     }
     
+    fileprivate func setupBackgroundPlayback() {
+        do {
+            try AVAudioSession.sharedInstance().setCategory(AVAudioSessionCategoryPlayback, with: .mixWithOthers)
+            print("AVAudioSession Category Playback OK")
+            do {
```

Comment:
You don't need another do here. you can put a second try statement inside the same do and whichever try fails first, it will be caught by the single catch

Reply:
This gets changed in the next PR, so I suggest we don't change it in this PR...I had a tricky time getting both background playback and remote control playback events working, and the approach being used in the next PR seems to work at the moment...